### DISCUSSION TITLE
Can I hide environment variables on terminal opening?

### DISCUSSION THREAD

**Discussion by jjrrGlobant** [vercel/hyper#8035] [Q&A]
Hi,

Everytime I open the terminal (even a new tab), I see some environment variables before the prompt:

![Screenshot 2024-09-05 at 09 21 12](https://github.com/user-attachments/assets/613b653f-0869-47c8-ba33-e771caecdaa9)

Is possible to avoid them to be shown ?

Thanks in advance,
Javi

**Comment by lnfel** [comment]
Hi, I am using version 3.4.1 and having no such issue. My assumption would be another config in your setup is dumping those lines on shell start-up. I recommend checking your shell config.
